src/main/kotlin/com/chattriggers/ctjs/api/message/TextComponent.kt

```diff
@@ -252,6 +262,26 @@ class TextComponent : Text {
         Message(this).actionBar()
     }
 
+    private fun parseItemContent(obj: Any?): HoverEvent.ItemStackContent? {
```

Comment:
This parameter will never be null, so lets remove the null marker here and in the `?.let`. Same for `parseEntityContent`.